src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/converter/FlowMapper.java

```diff
@@ -159,6 +160,42 @@ default FlowResponsePayload toFlowResponseOutput(FlowReadResponse r) {
     @Mapping(target = "type", ignore = true)
     FlowRequest toFlowRequest(FlowRequestV2 request);
 
+
+    @Mapping(target = "flowId", source = "id")
+    @Mapping(target = "sourceSwitch", expression = "java(request.getSource().getSwitchDpId())")
```

Comment:
You don't need eval expression here - instead, you can use `source = "source.switchDpId"`.

Reply:
fixed